modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/meetings/portlet/MeetingsPortlet.java

```diff
@@ -1198,6 +1193,13 @@ private long _getDurationInMinutes(CalendarBooking calendarBooking) {
 		return duration.abs().toMinutes();
 	}
 
+	private long _getDurationInMinutes(long startTime, long endTime) {
```

Comment:
since we expose this method, we should reuse it in the other method, something like:
`_getDurationInMinutes(calendarBooking.getEndTime() - calendarBooking.getStartTime())`

and, the return type should be `int`?